user_guide_src/source/models/model.rst

```diff
@@ -137,13 +138,15 @@ method.
 
 **$useSoftDeletes**
 
-If true, then any delete* method calls will simply set a flag in the database, instead of
+If true, then any delete* method calls will set ``deleted_at`` in the database, instead of
 actually deleting the row. This can preserve data when it might be referenced elsewhere, or
 can maintain a "recycle bin" of objects that can be restored, or even simply preserve it as
 part of a security trail. If true, the find* methods will only return non-deleted rows, unless
 the withDeleted() method is called prior to calling the find* method.
 
-This requires an INT or TINYINT field to be present in the table for storing state. The default field name is  ``deleted`` however this name can be configured to any name of your choice by using $deletedField property.
+This requires a DATETIME field to be present in the table for storing state. The default field
```

Comment:
Two things: 

1. requires either a datetime or integer field in the database as per their model's $dateFormat setting.

Okay - only one thing. :)

Reply:
Added!